tests/test_migrations.py

```diff
@@ -68,3 +71,15 @@ def model_label(self):
         Returns the label of the ``model``.
         """
         return self.model._meta.get_field('name')
+
+
+class MigrationTests(TestCase):
+    """
+    Runs migration tests using Django Command interface.
+    """
+    @override_settings(MIGRATION_MODULES={})
+    def test_migrations_are_in_sync(self):
+        out = StringIO()
+        call_command("makemigrations", dry_run=True, verbosity=3, stdout=out)
```

Comment:
I noticed there is also a `test_management.py` file as well, so I could potentially just put this in that file instead since I am in fact using the management command version. However, that file seems to be dedicated to _custom_ management commands, not the base Django commands.

Reply:
The management file is also where that test that asserts log order is... I'm ok with it living here!